pkgs/top-level/all-packages.nix

```diff
@@ -27541,6 +27541,8 @@ in
 
   ledger-live-desktop = callPackage ../applications/blockchains/ledger-live-desktop { };
 
+  lightning-pool = callPackage ../applications/blockchains/lightning-pool.nix { };
```

Comment:
```suggestion
  lightning-pool = callPackage ../applications/blockchains/lightning-pool { };
```

Reply:
Same here as with loop. This is not the way that lnd was done. I am happy to do as you ask but I was going based off of how I saw other related packages in the file tree.

Reply:
All other packages who do it like this in this file are packaged wrong.

Reply:
fixed by 8c9061b